packages/jellyfish-core/src/core.ts

```diff
@@ -0,0 +1,20 @@
+import * as methods from './methods'
+import {Client, Payload, JellyfishError} from './api'
+export {methods, Client, Payload, JellyfishError}
+
+/**
```

Comment:
Question, does this comment enables usage via JSDoc in VSC? Or it should have proper tags like @description, etc.

Reply:
For IntelliJ IDEA it does, not sure about VSCode.

Additionally for JSDoc exported symbols work well for JS but in the TS environment it's kind of irritating above all else.

So e.g. like @param, @return paired with @description are useful symbols but TypeScript is naturally strongly typed, which nuanced the need for export symbols. So people argued for the sake of brevity it should just be plain old text.

[my motivation was from here](https://github.com/denoland/deno/blob/main/docs/contributing/style_guide.md#use-jsdoc-for-exported-symbols)